### DISCUSSION TITLE
DNS conflict when adding a subdomain

### DISCUSSION THREAD

**Discussion by axgalache** [vercel/community#6492] [Help]
### Summary

I have a owned domain (ser-inclusivo.com). I have deployed a nextJS application on vercel, and I want that application to be in the subdomain "lf.ser-inclusivo.com". I have added that subdomain to vercel, and it says I have to add a CNAME DNS value with "cname.vercel-dns.com.". The problem is that I have previously confirgured icloud mail to that subdomain (lf.ser-inclusivo.com) and I am getting a conflict in my domain provider with the registries for the icloud mail in my domain provider (the registries for icloud mail are some TXT and MX registries) 

![CleanShot 2024-04-13 at 10 59 41](https://github.com/vercel/community/assets/3769641/e194cf82-97ae-4fa6-8820-dcaef6a3cbcc)

How can I solve this problem and add the subdomain successfully to vercel without conflicts with other DNS registries?

### Steps to Reproduce

- Existing subdomain configured with TXT and MX registries for apple icloud mail
- Try to add that subdomain (in my case lf.ser-inclusivo.com) to vercel and point to a branch
- Try to apply the CNAME "cname.vercel-dns.com." value in the third party domain provider webpage

**Comment by amyegan** [comment]
Hey @axgalache. You can read more about why having an MX record and CNAME records for the same hostname fails here: https://serverfault.com/a/18058

Instead of a CNAME, you could add an A record with the value `76.76.21.21` to use the subdomain with your project.
